creator-node/src/services/stateMachineManager/stateReconciliation/updateReplicaSet.jobProcessor.ts

```diff
@@ -123,7 +149,7 @@ const updateReplicaSetJobProcessor = async function ({
       replicaToUserInfoMap,
       enabledReconfigModes
     })
```

Comment:
what about specific errors from inside `_determineNewReplicaSet` - any of those worthwhile to record?

Reply:
bump

Reply:
There are errors from inside there that get bumped up already

Are there specific labels you had in mind?